### PR TITLE
[ODS-4286] Remove Net Framework Applications

### PR DESCRIPTION
**Pull request by tparandiev**

### CODE REVIEW

**Comment by semalaiappan** [comment]
@tparandiev - Build is failing now 

10:21:28
    Startup\ApiStartup.cs(14,12): error CS0246: The type or namespace name 'OwinStartup' could not be found (are you missing a using directive or an assembly reference?) [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
    Startup\AzureCloudOdsStartup.cs(15,12): error CS0246: The type or namespace name 'OwinStartupAttribute' could not be found (are you missing a using directive or an assembly reference?) [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
    Startup\AzureCloudOdsStartup.cs(15,12): error CS0246: The type or namespace name 'OwinStartup' could not be found (are you missing a using directive or an assembly reference?) [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
    Startup\ApiStartup.cs(14,42): error CS0246: The type or namespace name 'ApiStartup' could not be found (are you missing a using directive or an assembly reference?) [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
    Startup\AzureCloudOdsStartup.cs(15,48): error CS0246: The type or namespace name 'AzureCloudOdsStartup' could not be found (are you missing a using directive or an assembly reference?) [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
    Startup\ApiStartup.cs(12,17): error CS0234: The type or namespace name 'Owin' does not exist in the namespace 'Microsoft' (are you missing an assembly reference?) [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
    Startup\AzureCloudOdsStartup.cs(7,18): error CS0234: The type or namespace name 'Configuration' does not exist in the namespace 'System.Web' (are you missing an assembly reference?) [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
    Startup\AzureCloudOdsStartup.cs(12,17): error CS0234: The type or namespace name 'ApplicationInsights' does not exist in the namespace 'Microsoft' (are you missing an assembly reference?) [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
    Startup\AzureCloudOdsStartup.cs(13,17): error CS0234: The type or namespace name 'Owin' does not exist in the namespace 'Microsoft' (are you missing an assembly reference?) [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
    Startup\ApiStartup.cs(18,31): error CS0246: The type or namespace name 'StartupBase' could not be found (are you missing a using directive or an assembly reference?) [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
    Startup\ApiStartup.cs(20,33): error CS0115: 'ApiStartup.EnsureAssembliesLoaded()': no suitable method found to override [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
    Startup\AzureCloudOdsStartup.cs(19,41): error CS0246: The type or namespace name 'StartupBase' could not be found (are you missing a using directive or an assembly reference?) [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
    Startup\AzureCloudOdsStartup.cs(21,33): error CS0115: 'AzureCloudOdsStartup.ConfigurationSpecificRegistration()': no suitable method found to override [D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj]
10:21:28
      EdFi.Ods.WebService.Tests -> D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS\Application\EdFi.Ods.WebService.Tests\bin\Release\netcoreapp3.1\EdFi.Ods.WebService.Tests.dll
10:21:29
    WARNING: FAILED to build "D:\buildagent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\Ed-Fi-Ods.sln". 
10:21:29
    Please check the build log "D:\BuildAgent\temp\buildTmp\Ed-Fi-Ods.sln.msbuild.log" for details.
10:21:29
    Ensure that the Visual Studio SDK is installed.
10:21:29
    Build failed after 2m47s. Check the build log file 'D:\BuildAgent\temp\buildTmp\Ed-Fi-Ods.sln.msbuild.log' for errors.
10:21:29
    At D:\BuildAgent\work\904029b3fc87a31e\Ed-Fi-ODS-Implementation\Application\SolutionScripts\InitializeDevelopmentEnviro
10:21:29
    nment.psm1:296 char:13
10:21:29
    +             throw "Build failed after $(Get-DurationString $buildResu ...
10:21:29
    +             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
10:21:29
        + CategoryInfo          : OperationStopped: (Build failed af...og' for errors.:String) [], RuntimeException
10:21:29
        + FullyQualifiedErrorId : Build failed after 2m47s. Check the build log file 'D:\BuildAgent\temp\buildTmp\Ed-Fi-Od 
10:21:29
       s.sln.msbuild.log' for errors.

File: Application/EdFi.Ods.SandboxAdmin/Services/UserAccountManager.cs [hunk 1]
```diff
@@ -65,7 +65,7 @@ public async Task<CreateLoginResult> Create(CreateLoginModel model)
 
             return new CreateLoginResult
             {
-                Message = "A user already exists with this email address, but email address has not been confirmed.\n\n" +
+                Message = $"A user already exists with this email address, but email address has not been confirmed.{Environment.NewLine}{Environment.NewLine}" +
```
Comment (tbmeador-dlp) [inline Application/EdFi.Ods.SandboxAdmin/Services/UserAccountManager.cs:68]: Lets revert this and let ODS-4410 take care of it.

File: Application/EdFi.Ods.SandboxAdmin/Services/UserAccountManager.cs [hunk 2]
```diff
@@ -233,7 +233,7 @@ private async Task<CreateLoginResult> CreateNewUser(CreateLoginModel model)
             {
                 _log.Error("CreateNewUser", e);
 
-                var message = string.Format("An exception was thrown when attempting to create the user:\n{0}", e.Message)
+                var message = string.Format($"An exception was thrown when attempting to create the user:{Environment.NewLine}{{0}}", e.Message)
```
Comment (tbmeador-dlp) [inline Application/EdFi.Ods.SandboxAdmin/Services/UserAccountManager.cs:236]: Lets revert this and let ODS-4410 take care of it.

File: Application/EdFi.Ods.SandboxAdmin/Controllers/AccountController.cs [hunk 3]
```diff
@@ -357,7 +358,7 @@ private string[] GetFailedFieldsFromModelState()
         private string GetErrorMessageFromModelState()
         {
             var errorMessages = ModelState.SelectMany(x => x.Value.Errors.Select(error => error.ErrorMessage));
-            return string.Join("\n", errorMessages);
+            return string.Join(Environment.NewLine, errorMessages);
```
Comment (tbmeador-dlp) [inline Application/EdFi.Ods.SandboxAdmin/Controllers/AccountController.cs:361]: Lets revert this file and let ODS-4410 take care of it.

**Review by tbmeador-dlp — COMMENTED** [review]
Powershell changes look good but I haven't tested.

**Review by semalaiappan — APPROVED** [review]

**Pull request merged**